components/footer.tsx

```diff
@@ -8,13 +8,35 @@ const Footer = ({
       } lg:text-left`}
     >
       <div className="flex flex-col p-4 text-center text-white">
-        <span className=" text-white">
```

Comment:
I think it's a bit weird that copyright, contact, power by listing in column. Would it be better if it look like the footer of this github page